docker/hab2_tpsrl.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM fairembodied/habitat-challenge:habitat_rearrangement_2022_base_docker
+ADD agents/habitat_baselines_agent.py agent.py
+ADD configs/ /configs/
+RUN /bin/bash -c "mkdir -p data/models"
```

Comment:
Why not just add the whole `data/models` folder to this docker rather than adding independent files?